### ISSUE TITLE
Unintuitive error on trying to add nested folders in a single commit.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [salvatorelai/svnx#103]
```
What steps will reproduce the problem?
1. Open a remote repository
2. Click Make Dir
3. Type 'a' under directory, IF it does not exist, and click "Add".
4. Type 'a/b' under directory, and click "Add".
5. Type a commit message.
6. Click "Commit"

What is the expected output? What do you see instead?
I would be unsurprised by any of the following:
1. SvnX making two folders, one inside the other, on the repository.
2. An error letting me know I cannot create nested folders in one commit
3. An error letting me know "/" is not a valid folder character.
What actually happened:
If 'a' exists, the message: "Local, non-commit operations do not take a log 
message or revision properties." displays.
If 'a' does not exist, the message: Illegal repository URL " [one double quote]

What version of the product are you using? On what operating system?
MacOS 10.6.4, SvnX 

Please provide any additional information below.
The second folder is listed as "a:b" rather than "a/b".

```

Original issue reported on code.google.com by `vanc...@gmail.com` on 26 Jul 2010 at 6:29

**Comment by GoogleCodeExporter** [comment]
```
Sorry, also I am using SvnX 1.2.
```

Original comment by `vanc...@gmail.com` on 26 Jul 2010 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
SvnX does not currently support creating of intermediate directories.
[This feature was added to Subversion 1.5.0, but appears to have had a number 
of problems.]

In the Copy, Move & Make Dir sheets svnX attempts to clean up any names entered.
The '/' character is translated to ':' as the Finder displays ':' as '/' in 
file names.
Other illegal characters are translated to '-' & the name is restricted to 128 
characters in length.

However, (having just now looked into this) there is a (longstanding) bug in 
svnX that means that names containing ':' fail to create a valid URL.  Hence 
the error message: Illegal repository URL ''. [That's actually 
<single-quote><single-quote>.]

Currently you cannot create a repository directory with svnX if the new name 
contains a ':'.
This will be fixed in the next release.
Thanks for the report.
```

Original comment by `chris...@gmail.com` on 27 Jul 2010 at 6:53
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3b1.
```

Original comment by `chris...@gmail.com` on 16 Sep 2010 at 12:30
- Changed state: **Fixed**
